sections/expansion_content.tex

```diff
@@ -1,44 +1,68 @@
+\begin{itemize}
+  \item If any of the following occur during Combat with Neutral Units, discard the Neutral Unit Card and draw another Card in its place:
+    \begin{itemize}
+      \item if you draw the same Neutral Unit more than once,
+      \item if you draw any Unit you already have in your army,
+      \item if you draw any Unit from your Faction.
+    \end{itemize}
+  \item While you Recruit Neutral Units, you cannot Recruit any Units from a Faction that is controlled by a player or any Units that are already in any player's army.
+  Discard that Unit Card and draw another.
+\end{itemize}
+
 \subsection*{Permanent Cards}
-Added by multiple expansions, explained in \pagelink{Playerdecks}{Player Decks}.
+Cards with a new type of effect which gives players permanent advantages, explained in \pagelink{Playerdecks}{Player Decks}.
 
-\vspace*{1em}
-\begin{expansion}{rampart,cove}
-\subsection*{War Machines}
-  Explained in \pagelink{War Machines}{Deck-building}.
-\end{expansion}
+\subsection*{Schools of Magic}
+\pagelink{Schools of Magic}{Schools of Magic} gain more significance with the expansions, as there are now Card effects which use them, e.g.~your Heroes can specialize in certain Schools to make their Spells more powerful.
 
-\vspace*{1em}
-\begin{expansion}{inferno}
-\subsection*{Summoning}
-  Explained in the \pagelink{Summoning}{Units} section.
-\end{expansion}
+\subsection*{Unique Card effects with their own components}
+Several new Cards from various expansions come with their own Token components and bring complex effects to the game.
+These are all explained in the separate section \pagelink{Detailed Card Effects}{Detailed Card Effects}.
 
-\vspace*{1em}
-\begin{expansion}{inferno}
-\subsection*{Empowered Statistic Cards}
-  Explained with regular Statistic Cards in \pagelink{Empowered Statistic}{Deck-building}.
-\end{expansion}
+\subsection*{More of the same}
+In addition to new gameplay mechanics, expansions also expand the variety of existing components: new Factions, new Scenarios, new Spells, new Heroes, new Artifacts, and new Map Locations.
 
-\vspace*{1em}
-\begin{expansion}{inferno}
-  \subsection*{Random Town}\index{Random Town}
-  See it in \pagelink{Random Town}{All Map Locations}.
-\end{expansion}
+\begin{center}
+    \includegraphics[width=0.75\linewidth]{\images/tower-lineup.png}
+\end{center}
+\vspace*{\fill}
+\end{multicols*}
+\pagebreak
 
-\begin{expansion}[before=\vspace*{-11mm}]{fortress}
+\textbf{The following expansions each offer a new Faction and some new specific gameplay mechanics:}
+\begin{multicols}{2}
+\begin{expansion}[title=]{fortress}
+  \subsection*{\color{fortress}Fortress Expansion}
+  \setlength\intextsep{0pt}
+  \setlength\columnsep{0.8em}
+  \begin{wrapfigure}{l}{0.35\textwidth}
+      \includegraphics[width=\linewidth]{\boxcovers/fortress-box.png}
+  \end{wrapfigure}
+  Contains the \textbf{Fortress Faction}: poisonous swamp beasts under the lead of Witches and Beastmasters.\par
```

Comment:
Capitalize: `Poisonous swamp...`

Reply:
Ah, the flip-flop is my fault. I suggested "under command of" in one place but not in the others. @DerSchlupf, do you think we should change it everywhere?